### PR TITLE
5-1-updates

### PR DESCRIPTION
**Pull request by jenjoe22**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--prisma-cloud-docs--hlxsites.hlx.page/
- After: https://<branch>--prisma-cloud-docs--hlxsites.hlx.page/
- Worker: https://prisma-cloud-docs-production.adobeaem.workers.dev/?branch=<branch>

### CODE REVIEW

File: docs/en/enterprise-edition/content-collections/get-started/access-prisma-cloud.adoc [hunk 1]
```diff
@@ -69,12 +75,14 @@ If you see the serial number for your instance and want to change it to a descri
...
 === Log in Using the Prisma Cloud URL
 
-If you have accepted the EULA and have a Palo Alto Networks CSP account, you can access the Prisma Cloud app directly.
+Follow the steps below to access the Prisma Cloud app directly.
 
 [.procedure]
 . Access the Prisma Cloud App.
 +
-Enter your username and password. The URL is *<Prisma Cloud URL>*/auth/signin*
+Navigate to https://appX.prismacloud.io/login (where X is the instance where your tenant is provisioned, for example app3). Use the xref:#domain-list[domain-list] list above for domain specific URLs.
```
Comment (ftbrecordspan) [inline docs/en/enterprise-edition/content-collections/get-started/access-prisma-cloud.adoc:83]: @jenjoe22 can we change the formatting here to reflect that the URLs don't all follow the 'appx' convention?

Generally we follow a {BASE_URL}/login convention where the BASE_URL in question are those laid out in the linked doc.

File: docs/en/enterprise-edition/content-collections/get-started/access-prisma-cloud.adoc [hunk 2]
```diff
@@ -69,12 +75,14 @@ If you see the serial number for your instance and want to change it to a descri
...
+
+. Enter your username and password. 
 
 . *Sign in*.
 +
```
Comment (ftbrecordspan) [inline docs/en/enterprise-edition/content-collections/get-started/access-prisma-cloud.adoc:88]: The login page now offers multiple options here which I think we should disclose. In addition to the "Sign In" button, we have a "Sign in with SSO" button which is not discussed here. And we should probably cover the two distinct login flows behind the SSO button as well (OIDC, PANW-SSO).

File: docs/en/enterprise-edition/content-collections/get-started/access-prisma-cloud.adoc (hunk 1 above)
Comment (jenjoe22) [reply docs/en/enterprise-edition/content-collections/get-started/access-prisma-cloud.adoc:83]: Does this work @ftbrecordspan?

File: docs/en/enterprise-edition/content-collections/get-started/access-prisma-cloud.adoc [hunk 3]
```diff
@@ -69,14 +75,20 @@ If you see the serial number for your instance and want to change it to a descri
...
 === Log in Using the Prisma Cloud URL
 
-If you have accepted the EULA and have a Palo Alto Networks CSP account, you can access the Prisma Cloud app directly.
+Follow the steps below to access the Prisma Cloud app directly.
 
 [.procedure]
 . Access the Prisma Cloud App.
 +
-Enter your username and password. The URL is *<Prisma Cloud URL>*/auth/signin*
+Navigate to {BASE_URL}/.prismacloud.io/login. Use the xref:#domain-list[domain-list] list above for domain specific BASE_URLs.
```
Comment (ftbrecordspan) [inline docs/en/enterprise-edition/content-collections/get-started/access-prisma-cloud.adoc:83]: @jenjoe22 I believe we're looking for '{BASE_URL}/login' here

File: docs/en/enterprise-edition/content-collections/get-started/access-prisma-cloud.adoc (hunk 2 above)
Comment (jenjoe22) [reply docs/en/enterprise-edition/content-collections/get-started/access-prisma-cloud.adoc:92]: I updated with all the login options we're offering now. Please take a look @ftbrecordspan.

File: docs/en/enterprise-edition/content-collections/administration/setup-sso-integration-on-prisma-cloud/get-started-with-saml-sso/get-started-with-saml-sso.adoc [hunk 4]
```diff
@@ -9,11 +9,11 @@ Enable single sign-on (SSO) on Prisma Cloud using an Identity Provider (IdP) tha
...
 
-To enable SSO, you must first complete the setup on the IdP. Then, log in to Prisma Cloud using an account with System Admin privileges to configure SSO and redirect login requests to the IdP’s login page, so that your Prisma Cloud administrative users can log in using SSO. After you enable SSO, you must access Prisma Cloud from the IdP’s portal. Prisma Cloud supports IdP initiated SSO, and it’s SAML endpoint supports the POST method only.
+To enable SSO, you must first complete the setup on the IdP. Then, log in to Prisma Cloud using an account with System Admin privileges to configure SSO and redirect login requests to the IdP’s login page, so that your Prisma Cloud administrative users can log in using SSO. After you enable SSO, you must access Prisma Cloud from the IdP’s portal.
```
Comment (Aishbn) [inline docs/en/enterprise-edition/content-collections/administration/setup-sso-integration-on-prisma-cloud/get-started-with-saml-sso/get-started-with-saml-sso.adoc:12]: Replace "log in" with login in the second sentence. After Then...

**Review by Aishbn — APPROVED** [review]
@jenjoe22 - 2 minor comments to address. Rest LGTM.

**Pull request merged**